core/vm.cpp

```diff
@@ -1704,6 +1704,9 @@ class Interpreter {
             case json::value_t::discarded: {
                 abort();
             }
+
+            case json::value_t::binary:
```

Comment:
This should never happen (as this is a non-standard extension which doesn't occur when parsing JSON normally). I don't think just breaking in such case is reasonable. Let's have `abort()` here.